internal/graph/check.go

```diff
@@ -149,6 +151,13 @@ func (r *ResolveCheckRequest) GetContext() *structpb.Struct {
 	return nil
 }
 
+func (r *ResolveCheckRequest) GetConsistency() openfgav1.ConsistencyPreference {
+	if r != nil {
+		return r.Consistency
+	}
+	return openfgav1.ConsistencyPreference_MINIMIZE_LATENCY
```

Comment:
**Question:** I know this is the general default for the functionality but is `MINIMIZE_LATENCY` really the right setting to be returned here? Shouldn't this function instead be returning `UNSPECIFIED`?

Reply:
Additionally, codecov is indicating that L158 isn't covered. Is there a practical way of getting it tested?

Reply:
Yeah it probably should be returning `UNSPECIFIED`.

I'll try and see if we can test that line, I'm not certain if it's possible because I _think_ the `consistency` parameter is set to `UNSPECIFIED` if it's not included in the request.

Reply:
Yeah taking a look through under the debugger, if `consistency` is not set on a `ResolveCheckRequest` then it will be set to `UNSPECIFIED`. I believe this is because we set `UNSPECIFIED` as the zero value for `openfgav1.ConsistencyPreference`